scripts/getGitCommit.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if command -v git &>/dev/null && git rev-parse &>/dev/null; then
+    GIT_COMMIT=$(git rev-parse HEAD)
```

Comment:
Can you add these lines as well? 
        if [ -n "$(git status --porcelain --untracked-files=no)" ]; then
                GIT_COMMIT="$GIT_COMMIT-unsupported"
        fi

Reply:
Not adding it here because our current build leaves some untracked files